python/hail/dataset.py

```diff
@@ -2344,6 +2345,40 @@ def ld_prune(self, r2=0.2, window=1000000, memory_per_core=256, num_cores=1):
 
     @handle_py4j
     @requireTGenotype
+    def ld_matrix(self):
+        """Computes the linkage disequilibrium (correlation) matrix for the variants in this VDS.
+        
+        **Examples**
+        
+        >>> ld_mat = vds.ld_matrix()
+        
+        **Notes**
+        
+        Each entry (i, j) in the LD matrix gives the :math:`r` value between variants i and j, defined as 
+        `Pearson's correlation coefficient <https://en.wikipedia.org/wiki/Pearson_correlation_coefficient>`__ 
+        :math:`{\\rho}_{x_i,x_j}` between the two genotype vectors :math:`{\\mathbf{x_i}}` and :math:`{\\mathbf{x_j}}`.
+
+        .. math::
+
+            {\\rho}_{x,y} = \\frac{\\mathrm{Cov}(X,Y)}{\\sigma_X \\sigma_Y}
+            
+            
+        Also note that variants with zero variance (:math:`{\\sigma = 0}`) will be dropped from the matrix. 
+
+        .. caution::
+
+          The matrix returned by this function can easily be very large. Most likely you'll want to downsample or filter variants 
```

Comment:
I'd expand the first sentence to: The matrix returned by this function can easily be very large with most entries near zero (for example, entries between variants on different chromosomes in a homogenous population).